scRNA-seq/04-tag-based_scRNA-seq_processing.Rmd

```diff
@@ -93,13 +93,13 @@ This will take about 20 minutes to run, so we will start now, then talk about th
 
 ```
 salmon alevin -l ISR \
-  -i ~/shared-data/reference/refgenie/mm10_cdna/salmon_index/short/short/ \
-  -1 data/tabula-muris/fastq-raw/10X_P4_3/10X_P4_3_L001_R1_001.fastq.gz \
-  -2 data/tabula-muris/fastq-raw/10X_P4_3/10X_P4_3_L001_R2_001.fastq.gz \
+  -i ~/shared-data/reference/refgenie/mm10_cdna/salmon_index/short \
+  -1 data/tabula-muris/fastq/10X_P4_3/10X_P4_3_L001_R1_001.fastq.gz \
+  -2 data/tabula-muris/fastq/10X_P4_3/10X_P4_3_L001_R2_001.fastq.gz \
   --chromium  \
   -p 5 \
   -o data/tabula-muris/alevin-quant/10X_P4_3 \
-  --tgMap data/tabula-muris/mm_ensdb95_tx2gene.tsv \
+  --tgMap index/Mus_musculus/Mus_musculus.GRCm38.95.versioned_tx2gene.tsv \
```

Comment:
I forgot that even switching to tximeta will not get rid of the necessity for the tx2gene.tsv (unless tximeta has a nice seamless way to make this file for us?)

Reply:
I don't think it can be avoided for single cell mapping, as that is outside the R ecosystem. But there might be a way to update this when we get there. Right now I just wanted it to work with a more parallel directory layout.